pkg/status/status.go

```diff
@@ -408,3 +408,15 @@ func (s *status) monitorClusterStatus() {
 		}
 	}
 }
+
+// ReportMigration sets the clusterOperator status to signal that migration logic is in progress,
+// while upgrading from openshift 4.1.x to openshift 4.2.0
+func ReportMigration(cfg *rest.Config, mgr manager.Manager, namespace string, version string, stopCh <-chan struct{}) error {
+	instance = new(cfg, mgr, namespace, version, stopCh)
+	conditionListBuilder := clusterStatusListBuilder()
+	conditionListBuilder(configv1.OperatorProgressing, configv1.ConditionTrue, fmt.Sprintf("Performing migration logic to progress towards release version: %s", instance.version), "Upgrading from 4.1 to 4.2")
```

Comment:
You cannot hard code the version in the reason. Moreover the reason has to be one CamelCase word, so you could just go with `Upgrading`

Reply:
I wasn't aware of that thanks for pointing this out!